howto/install/high-availability.md

```diff
@@ -38,19 +38,16 @@ The port to use is always `8444`, the same AMS is listening on.
 
 ## Anbox Streaming Stack
 
-[note type="caution" status="Warning"]If you are upgrading from 1.4 or earlier, check [the upgrade instructions](https://discourse.ubuntu.com/t/upgrading-from-previous-versions/17750).[/note]
+If you are upgrading from 1.4 or earlier, check [the upgrade instructions](https://discourse.ubuntu.com/t/upgrading-from-previous-versions/17750) before proceeding.
```

Comment:
I think we can remove that comment, as 1.4 is very very old now.

Reply:
Agree!